go/vt/tabletserver/planbuilder/dml.go

```diff
@@ -29,6 +29,11 @@ func analyzeUpdate(upd *sqlparser.Update, getTable TableGetter) (plan *ExecPlan,
 		return nil, err
 	}
 
+	// Store the WHERE clause as string for the transaction serializer.
+	buf := sqlparser.NewTrackedBuffer(nil)
+	buf.Myprintf("%v", upd.Where)
```

Comment:
I've looked into the fairness of waiting Go routines.

If there's any delay and a notion of order among waiting Go routines, they are released in the order they were "queued": https://play.golang.org/p/j4F8g4eCGO
(But if there's no delay, Go routines are woken up in a random order because their queue order was probably random to begin with?)

That's great. It means there's no downside to using only `select` (without a dedicated queue) for our use case.